docs/src/main/asciidoc/se/webserver.adoc

```diff
@@ -389,6 +472,125 @@ include::{sourcedir}/se/WebServerSnippets.java[tag=snippet_21, indent=0]
 * Otherwise, the exceptions are translated to an Internal Server Error HTTP
 error code `500`.
 
+=== Configuration Options
+
+include::{rootdir}/config/io_helidon_common_tls_Tls.adoc[leveloffset=+2,tag=config]
+
+== Server Features
+Server features provide additional functionality to the WebServer, through modification of the server configuration,
+listener configuration, or routing.
+
+A server feature can be added by implementing `io.helidon.webserver.spi.ServerFeature`.
+Server features support automated discovery, as long as the implementation is available through Java `ServiceLoader`.
+Server features can also be added through configuration, as can be seen above in <<Configuration Options, Configuration Options>>,
+configuration key `features`.
+
+All features (both `ServerFeature` and `HttpFeature`) honor weight of the feature (defined either through `@Weight` annotation,
```

Comment:
There's no discussion of `HttpFeature`s specifically which might be OK but people considering adding a feature, reading this, and seeing `HttpFeature` mentioned will probably wonder what the difference is and how to decide which type of feature they should implement for their use case. It would be good if the doc had a very brief explanation of the difference between then and how to choose. 

Maybe if this topic needs a more complete treatment (maybe throwing `HttpService` into the mix as well?) that should be done in a separate PR.

Reply:
Updated documentation of Routing - added a short description of `HttpFeature`